### ISSUE TITLE
Is HighwayHash available for Java?

### ISSUE THREAD

**Issue by kant111** [google/highwayhash#72]
Is sha256 highwayhash available for Java? any benchmarks compared C or C++ counterpart?

**Comment by jan-wassenberg** [comment]
Hi, the [Java port](https://github.com/google/highwayhash/blob/master/java/com/google/highwayhash/HighwayHash.java) does indeed support 256-bit HWH.

**Comment by kant111** [comment]
Thanks! I tried it and I got 2X more performance Java's MessageDigest for sha256. 

My data size is 2KB and the performance numbers are as follows:

HighWayHash hash256: 225MB/sec
MessageDigest SHA-256: 115MB/sec

I read [here](https://github.com/minio/highwayhash) that highway hash should not be used if strong collision resistance is required.

so I am wondering if there is a Java library similar to [this](https://github.com/minio/sha256-simd) that can leverage SSE or AVX instructions?
